implementation/src/test/java/io/smallrye/reactive/adapt/UniAdaptFromTest.java

```diff
@@ -115,21 +115,21 @@ public void testCreatingFromAFlowableWithFailure() {
 
     @Test
     public void testCreatingFromAMono() {
-        Uni<Integer> uni = Uni.createWith(new FromMono<Integer>()).from(Mono.just(1));
+        Uni<Integer> uni = Uni.createWith(new FromMono<>(), Mono.just(1));
```

Comment:
Currently `Mono` and `Flux` converters are only in the test packages so that the main implementation doesn't need to depend on Reactor

Reply:
Yes we need an additional module to ship them.

Reply:
Ultimately it would need to be in its own module too. However until we get rid of the dependency, it would work that way.

I would prefer having is in a separate module immediately, so we don't break later.